traffic_ops/bin/traffic_ops_ort.pl

```diff
@@ -783,6 +771,53 @@ sub get_print_current_client_connections {
 	( $log_level >> $DEBUG ) && print "DEBUG There are currently $current_connections connections.\n";
 }
 
+sub get_update_status {
+	my $status_api;
+	my $uri     = "/api/1.3/servers/$hostname_short/update_status";
```

Comment:
I don't understand why this route would not exist (/api/1.3/servers/$hostname_short/update_status). If it doesn't exist, then TO needs to be upgraded so it does exist. By falling back to /update/$hostname_short  you are going to have major issues when /update/$hostname_short is eventually deprecated. 

IMO "UI" routes like /update/$hostname_short need to be removed from ORT entirely and ORT needs to leverage real API endpoints or it runs the risk of not working at all as TO evolves away from the UI namespace.

Reply:
ORT is not always updated in concert with traffic ops, so the thinking here is to maintain compatibility in both directions, at least for a little while.  Because the 1.3 api did not originally have the field we need to determine instant invalidation, we need that fallback ability.  The only other solution I can think of is to a) increase the api version of that endpoint to 1.4 or b) look for that field within the endpoint and then fail if the new field isn't present.  But ORT will not work without either the old UI endpoint or this fixed 1.3 endpoint at this time.